### ISSUE TITLE
Refactor curriculum notes by lesson instead of by instructor?

### ISSUE THREAD

**Issue by monicathieu** [ColumbiaSWC/guidebook#1]
Currently, the [curriculum notes page](../blob/master/docs/curriculum-notes.md) is organized with notes by instructor (presumably the way the notes were originally organized before being put together into one big page). The page might be a bit easier to navigate if the content were re-organized such that the highest header level corresponded to lesson, so that if multiple instructors have complementary notes on the same lesson, all that info gets to be together. Within lesson, where appropriate, notes could then be separated by instructor, or combined if multiple instructors agree on a particular note.

**Comment by smythp** [comment]
Yes, this is a bit of a placeholder as it is now. I'll refactor it as you suggest, which will be more practical for instructors trying to use it to find advice on their lesson.
